app/src/main/res/values/strings.xml

```diff
@@ -101,26 +102,26 @@
     <string name="confirm_create">Confirm create</string>
     <string name="from">From</string>
     <string name="to">To</string>
-    <string name="pending" translatable="false">Pending</string>
-    <string name="accepted" translatable="false">Accepted</string>
-    <string name="rejected" translatable="false">Rejected</string>
-    <string name="forwarded" translatable="false">Forwarded</string>
+    <string name="pending">Pending</string>
+    <string name="accepted">Accepted</string>
+    <string name="rejected">Rejected</string>
+    <string name="forwarded">Forwarded</string>
     <string name="leave_type">Leave type</string>
     <string name="compensation_time">Compensation time</string>
     <string name="example_time_from">Ex: 2017/05/20 17:30</string>
     <string name="example_time_to">Ex: 2017/05/20 18:30</string>
     <string name="confirm_request_leave">Confirm request leave</string>
     <string name="submit">Submit</string>
     <string name="next">Next</string>
-    <string name="status" translatable="false">Status</string>
-    <string name="all" translatable="false">All</string>
-    <string name="month" translatable="false">Month</string>
-    <string name="date" translatable="false">Date:</string>
-    <string name="date_of_creation" translatable="false">Date of creation:</string>
-    <string name="day_session" translatable="false">Day session</string>
-    <string name="am" translatable="false">am</string>
-    <string name="pm" translatable="false">pm</string>
-    <string name="time_request" translatable="false">Time request:</string>
+    <string name="status">Status</string>
```

Comment:
FATAL: [MissingTranslation] "`status`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)